nengo_gui/namefinder.py

```diff
@@ -22,9 +22,12 @@ def find_names(self, net):
         classes = (nengo.Node, nengo.Ensemble, nengo.Network,
                    nengo.Connection)
 
+        private = False
+        in_lists = False
```

Comment:
Because it's bad to initialise variables inside a loop? I can't find a reference for this however...